src/pages/proposals/create/StepProposalType.vue

```diff
@@ -82,38 +92,78 @@ export default {
 
     referenceComponent () {
       let result = null
+      const options = this.memberType === MEMBER_TYPE.COMMUNITY ? this.config.types.community.options : this.memberType === MEMBER_TYPE.CORE ? this.config.types.core.options : this.config.options
       if (this.selection) {
         let found = false
-        Object.values(this.config.options).forEach((opt) => {
+        Object.values(options).forEach((opt) => {
           if (!found) {
-            if (opt.key === this.selection) return null
-
-            if (opt.options[this.selection]) {
-              result = opt.options[this.selection].options
-              found = true
+            if (this.selection === opt.key) {
+              if (opt.options && typeof (opt.options) === 'string') {
+                result = opt.options
+                found = true
+              }
+            } else {
+              if (opt.key === this.selection) return null
+              if (opt.options[this.selection]) {
+                result = opt.options[this.selection].options
+                found = true
+              }
             }
           }
         })
       }
-
       return result
-    }
+    },
+
+    type () { return this.selection ? this.selection.toUpperCase() : null }
   },
 
   methods: {
     selectOption (option) {
       this.$emit('select', option)
       if (this.$q.screen.gt.md) {
-        switch (option) {
-          case 'contribution':
-            this.$emit('next')
-            break
-          case 'archetype':
-            this.$emit('next')
-            break
-          case 'obadge':
-            this.$emit('next')
-            break
+        if (this.memberType === MEMBER_TYPE.COMMUNITY) {
+          switch (option) {
+            case 'poll':
+              this.$emit('next')
+              break
+            case 'payout':
+              this.$emit('next')
+              break
+          }
+        } else if (this.memberType === MEMBER_TYPE.CORE) {
+          switch (option) {
+            case 'quest':
+              // this.$emit('next')
+              break
+            case 'obadge':
```

Comment:
@arsenijesavic this doesn't look like a good place to use switch. I see that every option ( except quest ) is returning `this.$emit('next')`